### ISSUE TITLE
GeoTIFF reader and writer

### ISSUE THREAD

**Issue by ebocher** [orbisgis/h2gis#571]
Implement a Geotiff driver to read and write GeoTIFF file.

**Comment by nicolas-f** [comment]
Beam have GeoTiff related class:
https://github.com/bcdev/beam/blob/master/beam-geotiff/src/main/java/org/esa/beam/dataio/geotiff/GeoTiffProductReader.java

**Comment by ebocher** [comment]
Yes but Beam uses Geotools to parse geotiff fields.

**Comment by nicolas-f** [comment]
Ok. Used specification is here http://www.remotesensing.org/geotiff/spec/contents.html

Does gdms was able to read GeoTiff in the past ?

**Comment by ebocher** [comment]
No. GRAP was used to read and write acsii and world file image. 
I propose to test raster datatype with a basic model :  tiff + wordlfile + prj (optional).
